core/invoker/src/main/scala/org/apache/openwhisk/core/invoker/Invoker.scala

```diff
@@ -250,7 +251,7 @@ trait InvokerCore {
   def disable(): String
   def isEnabled(): String
   def backfillPrewarm(): Route
-  def status(): Future[Map[String, List[String]]]
+  def getPoolState(): Future[Either[NotSupportedPoolState, TotalContainerPoolState]]
```

Comment:
This is one small change from the original codes.
I wanted to isolate Akka-HTTP dependency on the invoker server layer.
So each InvokerReactive just returns non-Akka-HTTP-related data.
The invoker server is only dependent on Akka-HTTP.